client/src/components/screen/ProfileUpdate.tsx

```diff
@@ -121,42 +137,48 @@ const Screen: FC<Props> = () => {
     }
   };
 
-  const uploadImage = async (uri: string): Promise<string> => {
+  const uploadImage = (uri: string): void => {
     const fileName = uri.split('/').pop() || '';
     const fileTypeMatch = /\.(\w+)$/.exec(fileName);
     const fileType = fileTypeMatch ? `image/${fileTypeMatch[1]}` : 'image';
-    const data: FormData = new FormData();
-    const token = await AsyncStorage.getItem('token');
-    data.append('profile', {
-      uri: uri,
+    const file = new Blob([uri], {
       type: fileType,
-      name: fileName,
+      endings: 'native',
     });
```

Comment:
Wow so you found the way to do this directly without `FormData`?

Reply:
Not really. I just replace using the plain fetch function here to use the mutation then use the GraphQL fetch.
But I needed to create a `file` to send as a parameter value (`mutationConfig.uploadables.file`) on the mutation, which I choose to do it with the Blob object. (See: https://github.com/jaydenseric/apollo-upload-client/#Blob).
Then this file is appended on the `FormData` from the `GraphQL FetchFunction` (see `fetch.ts`)